src/classes/UTIL_Http_TEST.cls

```diff
@@ -307,13 +306,4 @@ public with sharing class UTIL_Http_TEST {
         Test.setMock(HttpCalloutMock.class, callout);
     }
 
-    public static void mockChargeElevateBatch(String responseBody){
-        CalloutMock callout = new CalloutMock()
-                .withStatusCode(UTIL_Http.STATUS_CODE_OK)
```

Comment:
I like the move to a single method, were these Status values just never being checked?

Reply:
These status values are the default values in the constructor.